inconspiquous/dialects/qu.py

```diff
@@ -51,6 +67,37 @@ def num_qubits(self) -> int:
         return 1
 
 
+@irdl_op_definition
```

Comment:
It would be great if each operation could have:
- a short docstring
- an `__init__` method
- an `assembly_format` string.